src/Panel/Titlebar/Titlebar.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Titlebar.less';
+
+/**
+ * Class representating the titlebar. Usually used in a panel.
+ *
+ * @class The TitleBar
+ * @extends React.Component
+ */
+export class Titlebar extends React.Component {
+
+  /**
+   * The properties.
+   * @type {Object}
+   */
+  static propTypes = {
+    /**
+     * The className which should be added.
+     * @type {String}
+     */
+    className: PropTypes.string,
+    /**
+     * The children to show in the Window.
+     * @type {node}
+     */
+    children: PropTypes.node,
+    /**
+     * The dispatch function.
+      @type {Function}
+     */
+    dispatch: PropTypes.func,
+    /**
+     * The object of the parent container of the titlebar class.
+     * @type {object}
+     */
+    parent: PropTypes.object,
+    /**
+     * Additional elements display besides the collapseButton.
+     * @type {Array<object>}
+     */
+    tools: PropTypes.arrayOf(PropTypes.element),
+    /**
+     * Whether to allow collasping or not.
+     * @type {boolean}
+     */
+    collapsible: PropTypes.bool,
+    /**
+     * Whether to allow closing or not.
+     * @type {boolean}
+     */
+    closable: PropTypes.bool,
+
+    /**
+     * The tooltip of the compress button.
+     * @type {String}
+     */
+    compressTooltip: PropTypes.string,
+
```

Comment:
remove empty line to define props consistently (below `closable` and `closeTooltip` as well)